HOWTO_RELEASE

```diff
@@ -0,0 +1,45 @@
+How to issue a GeoUtils release.
+
+ 1. Go to your local main repository (not the fork) and ensure your master branch is synced:
+       git checkout master
+       git pull
+ 2. Look over whats-new.rst and the docs. Make sure "What's New" is complete
+    (check the date!) and add a brief summary note describing the release at the
+    top.
+ 3. If you have any doubts, run the full test suite one final time!
+      pytest --run-slow --mpl .
```

Comment:
Can't get this to run on my machine. Does it require a specific pytest version?

Reply:
me neither. Can't find out anything useful by googling. @fmaussion , any tips?

Reply:
`--mpl` is for matplotlib image tests, you probably don't need that.